src/export/StructureDefinitionExporter.ts

```diff
@@ -80,6 +81,20 @@ export class StructureDefinitionExporter {
             element.bindToVS(rule.valueSet, rule.strength as ElementDefinitionBindingStrength);
           } else if (rule instanceof ContainsRule) {
             rule.items.forEach(item => element.addSlice(item));
+          } else if (rule instanceof CaretValueRule) {
+            if (rule.path !== '') {
+              element.setInstancePropertyByPath(
+                rule.caretPath,
+                rule.value,
+                this.resolve.bind(this)
+              );
+            } else {
```

Comment:
Do we provide a way to access the root element?  I expect that when `rule.path === ''`, then the `element` received in line 68 is probably the root element, but we toss that value and interpret it as setting something on the structure definition.  That's how we've been thinking about it all along (so you've implemented it right), but I'm thinking we need a way to set something on the root as well.  Have we discussed this before?  Any ideas?  Maybe if path is `$root` (or `$rootElement`)?  Actually the [FHIR FHIRPath implementation](http://hl7.org/fhir/R4/fhirpath.html) uses `%` for special variables, so if we wanted to conform to that... then `%rootElement`?

Reply:
I spoke w/ @ngfreiter about this yesterday and we agreed that we *should* support it somehow, but that designing and implementing that support should be separate from this PR.